saleor/graphql/schema.graphql

```diff
@@ -2328,6 +2328,19 @@ type LanguageDisplay {
   language: String!
 }
 
+type LimitInfo {
+  currentUsage: Limits!
+  maximumUsage: Limits!
```

Comment:
This should be `allowedUsage`, `usageLimit`, or something similar. It is not maximum usage, which would be the highest usage observed.